modules/api/src/main/java/org/apache/ignite/table/TableView.java

```diff
@@ -351,4 +353,9 @@
      */
     @NotNull <T extends Serializable> CompletableFuture<Map<R, T>> invokeAllAsync(@NotNull Collection<R> keyRecs,
         InvokeProcessor<R, R, T> proc);
+
+    /**
+     * @return Current transaction or null if a table is not enlisted in any transaction.
+     */
+    @Nullable Transaction transaction();
```

Comment:
Does it make sense to have tx() method in addition (or maybe rename this) just for short?

Reply:
Method abbrevs are forbidden.